play-services-location/core/provider/src/main/kotlin/org/microg/gms/location/network/wifi/MovingWifiHelper.kt

```diff
@@ -213,6 +213,29 @@ class MovingWifiHelper(private val context: Context) {
         }
         return location
     }
+    
+    private fun parseSncf(location: Location, data: ByteArray): Location {
+            val json = JSONObject(data.decodeToString())
+            if(json.getInt("fix") == -1) { // no fix
+            	return location
```

Comment:
Please throw an exception in this case. Returning the default location object might cause problematic behavior.

Reply:
Done!